PostgreSQL.md

```diff
@@ -6,6 +6,12 @@ Development
 -----------
 ### Installing
 
+Prepare on Ubuntu:
+```
+sudo apt-get install liblzo2-dev
+export CGO_LDFLAGS='-no-pie'
```

Comment:
I'm not sure this
`export CGO_LDFLAGS='-no-pie'`
is useful and can be recommended.
Looks like travis compiles wal-g without this?